packages/babel-macro-react-rxjs/src/index.js

```diff
@@ -0,0 +1,58 @@
+const { createMacro } = require('babel-plugin-macros')
+const { default: template } = require('@babel/template');
+
+const buildTagWrapper = template(`
+  IMPORTED.addDebugTag(NAME, ID)(EXPRESSION)
+`);
```

Comment:
Cool helper from babel so we don't need to manually build this MemberExpression + CallExpression x2. Idea is we'll use this to replace `EXPRESSION` (the observable in `bind`) for all of this... which is just avoiding dealing with .pipe